django/conf/locale/th/formats.py

```diff
@@ -9,13 +9,25 @@
 MONTH_DAY_FORMAT = 'j F'
 SHORT_DATE_FORMAT = 'j M Y'
 SHORT_DATETIME_FORMAT = 'j M Y, G:i'
-# FIRST_DAY_OF_WEEK =
+FIRST_DAY_OF_WEEK = 0  # Sunday
 
 # The *_INPUT_FORMATS strings use the Python strftime format syntax,
 # see http://docs.python.org/library/datetime.html#strftime-strptime-behavior
-# DATE_INPUT_FORMATS =
-# TIME_INPUT_FORMATS =
-# DATETIME_INPUT_FORMATS =
+DATE_INPUT_FORMATS = [
+    '%d/%m/%Y',  # 25/10/2006
+    '%d %b %Y',  # 25 ต.ค. 2006
+    '%d %B %Y',  # 25 ตุลาคม 2006
```

Comment:
Thai calendar (Buddhist Era) is also a Gregorian calendar, but we start counting 543 years earlier than Common Era.

I think it wouldn't be hard to convert B.E. to C.E. A simple heuristic I used is that if the input year is over 2400 it should be B.E. (The current year is 2560) However, I think this is out of scope of this PR and implementing such system would require a lot more discussion.